x-pack/plugins/infra/public/components/logging/log_minimap/search_marker_tooltip.tsx

```diff
@@ -41,7 +43,11 @@ export class SearchMarkerTooltip extends React.PureComponent<SearchMarkerTooltip
                   }}
                   ref={measureRef}
                 >
-                  {children}
+                  <div
+                    className="euiToolTip__arrow"
+                    style={{ left: width || 0, top: (height || 0) / 2 - POPOVER_ARROW_SIZE / 2 }}
```

Comment:
I'll check it. Thanks! 👍

Reply:
I did check it and the methods are very different. The new one takes the HTMLElement of the anchor and the popover, which we don't track right now.

I'm going to merge it as it is. To be honest I think the best solution would be to allow `<EuiPopover>` to use any element as an anchor instead of forcing a `<span>`. That way we can use it on an SVG.